web/src/engine/websites/Akuma.ts

```diff
@@ -0,0 +1,47 @@
+import { Tags } from '../Tags';
+import icon from './Akuma.webp';
+import { Chapter, DecoratableMangaScraper, type Manga } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+const pageScript = `
+    new Promise( (resolve, reject) => {
+        try {
+            const url = window.location.href.replace(/\\/1$/, '');
+            const token = document.querySelector('meta[name="csrf-token"]').content.trim();
+            fetch(url, {
+                credentials: 'include',
+                headers: {
+                    'X-CSRF-TOKEN': token,
+                    'X-Requested-With': 'XMLHttpRequest',
+
+                },
+                 origin: window.location.origin,
+                 referrer: window.location.href,
+                 method: 'POST',
+            }).then(response => response.json()
+                .then(data => resolve(data.map(image => img_prt + '/' + image))));
```

Comment:
Either use the `.catch()` fluent method to reject the promise, or better use `async ... await` for `fetch`.